### DISCUSSION TITLE
Ransomeware in wiki

### DISCUSSION THREAD

**Discussion by Zinvoke** [polybar/polybar#2862] [Q&A] [answered]
A user edited the wiki and added links to malware for windows so I removed it atm anyone can edit the wiki here is what it was before I updated it:
[ed7b3f7](https://github.com/polybar/polybar/wiki/Home/ed7b3f733f19728ec7ed7f12c1721a662ce3cfa1)

**Comment by patrick96** [comment, marked as answer]
Thanks for the quick reaction :)

I want the wiki to be editable so people can add clarifications without having to go through posting something here first. I normally review wiki edits, but it may take me a day or so to look at them.

I blocked the user from polybar and reported them.

We never really had issues with people maliciously editing the wiki (this is only the second time I think). If things ramp up, I may have to consider making it read-only (unfortunately you can't lock only certain pages).

**Reply by Zinvoke** [reply]
Yeah no problem it was only ransomware pretending to be polybar ported for windows LOL I was like what let me try that out and windows defender was like nope.

**Reply by patrick96** [reply]
Lucky that defender detected that

**Comment by Zinvoke** [comment]
The infected files: 
https://github.com/AhmedAboSamra/VideoCaptureUtility/releases/tag/42

He is targeting other wiki's like:
https://github.com/alacritty/alacritty/wiki
